steward/src/commands/orchestrator/start.rs

```diff
@@ -16,10 +16,10 @@ use gravity_bridge::orchestrator::main_loop::{
 use gravity_bridge::relayer::main_loop::LOOP_SPEED as RELAYER_LOOP_SPEED;
 use std::{cmp::min, sync::Arc};
 
-/// Start Orchestrator
+/// Start the Orchestrator
 #[derive(Command, Debug, Parser)]
 #[clap(
-    long_about = "DESCRIPTION \n\n Start the Orchestrator in Sommelier Chain via the Gravity Bridge.\n This command loads a Cosmos and Ethereum key with their keyname from the keystore.\n It also takes an Orchestrator_only field which when set to true, starts the Orchestrator only\n without the relayer and when set to false, starts the Orchestrator with the relayer."
+    long_about = "DESCRIPTION\n\nStarts the Gravity Bridge Orchestrator and optionally the Relayer with the --relay flag.\nYou must provide the keys delegated by your validator node for Cosmos and Ethereum signing."
```

Comment:
With the adjusted language from the original, we should be specific that we mean key names and not the keys themselves.